packages/core/medusa-test-utils/src/medusa-test-runner-utils/bootstrap-app.ts

```diff
@@ -77,9 +77,7 @@ export async function startApp({
       })
 
     // TODO: fix that once we find the appropriate place to put this util
-    const {
-      GracefulShutdownServer,
-    } = require("@medusajs/medusa/dist/utils/graceful-shutdown-server")
+    const { GracefulShutdownServer } = require("../graceful-shutdown-server")
```

Comment:
Can't we just do `import` now that we removed the @medusajs/medusa dependency?

Reply:
👍